src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -754,8 +754,12 @@ do so, use `canMakePayment()`.
       request.show();
     }
 
-Note: This is only optional. Users can still add a new payment method in the
-Payment Request UI in general.
+Warning: In order to protect user privacy, calling <code>canMakePayment()</code>
+too often within a short time range causes [a "Quota exceeded"
```

Comment:
Can we provide any details of when this is thrown?

Avoiding details will likely cause developers discomfort - there is a phantom error which might appear but they don't know when or how.